src/components/ModalPayment/SquareModal.tsx

```diff
@@ -6,47 +6,42 @@ import 'react-square-payment-form/lib/default.css'
 import styles from './styles.module.scss';
 import SubmissionButton from './SubmissionButton';
 import {makeSquarePayment, SquarePaymentParams, Buyer} from "../../utilities/api"
-import ModalConfirmation from '../ModalConfirmation';
+
+import { useModalPaymentState, useModalPaymentDispatch } from "../../utilities/hooks/ModalPaymentContext/context"
+import { SET_MODAL_VIEW } from "../../utilities/hooks/ModalPaymentContext/constants"
```

Comment:
Though we should probably look at a way to avoid doing all this stuff with relative paths, there are packages available